src/action.cpp

```diff
@@ -693,6 +700,12 @@ ActionMkSort::generate()
     }
     break;
 
+    case SORT_FF:
+    {
+      run(SORT_FF, "17");
```

Comment:
Is there a specific reason why this is fixed to 17? Why not using an arbitrary value here?

Reply:
That might find more bugs.
But, it would require a sampler for prime numbers.
This was already useful.

Reply:
We could maybe for now specify a list of e.g. 10 prime numbers that get randomly picked?